{
  try {
    testSame("z==8, z++;");
    fail("Should raise an exception");
  }
 catch (  RuntimeException e) {
  }
}
